docs/content/release_notes/csi_rn_limitations.md

```diff
@@ -29,4 +29,11 @@ The following limitations apply when expanding volumes with the IBM block storag
 
 -   When using the CSI driver with IBM Spectrum Virtualize Family and IBM DS8000 Family products, during size expansion of a PersistentVolumeClaim (PVC), the size remains until all snapshots of the specific PVC are deleted.
 -   When expanding a PVC while not in use by a pod, the volume size immediately increases on the storage side. PVC size only increases, however, after a pod begins to use the PVC.
--   When expanding a filesystem PVC for a volume that was previously formatted but is now no longer being used by a pod, any copy or replication operations performed on the PVC (such as snapshots or cloning, and so on) results in a copy with the newer, larger, size on the storage. However, its filesystem has the original, smaller, size.
\ No newline at end of file
+-   When expanding a filesystem PVC for a volume that was previously formatted but is now no longer being used by a pod, any copy or replication operations performed on the PVC (such as snapshots or cloning, and so on) results in a copy with the newer, larger, size on the storage. However, its filesystem has the original, smaller, size.
+
+## Volume replication limitations
+
+The following limitations apply when using volume replication (remote copy) with the IBM block storage CSI driver:
+
+- The volume replication function is only supported for use with IBM Spectrum Virtualize Family storage systems.
```

Comment:
is this considered a limitation? because I assume we will add support for other storage systems in the future, and not because of any "external" functional limitation (only time limitation)

Reply:
you are correct. removed as limitation.